src/database.js

```diff
@@ -1,18 +1,26 @@
 const mongoose = require('mongoose');
 const bcrypt = require('bcrypt');
+const joi = require('joi');
 const util = require('./util');
 const { PNID } = require('./models/pnid');
 const { Server } = require('./models/server');
-const { mongoose: mongooseConfig } = require('../config.json');
-const { uri, database, options } = mongooseConfig;
+const config = require('../config.json');

```

Comment:
not a comment about this line, but if you're cleaning up account anyway.
want to add an example config? last time I had to get this running I had to brute force which config settings it required. (run, check error, add config, run check error add config, run, etc...)

Reply:
There should be one already? Is it outdated?

Reply:
didnt see it, it was in the source folder. my bad